#!/usr/bin/env python

PACKAGE = "touch_teleoperation"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("feeback_force_mode", str_t, 0, "Feedback force mode", "absolute")
gen.add("absolute_force_scale_x", double_t, 0, "Absoulte feedback force scale x", 1.0, -10.0, 10.0)
gen.add("absolute_force_scale_y", double_t, 0, "Absoulte feedback force scale y", 1.0, -10.0, 10.0)
gen.add("absolute_force_scale_z", double_t, 0, "Absoulte feedback force scale z", 1.0, -10.0, 10.0)
gen.add("absolute_force_scale_rx", double_t, 0, "Absoulte feedback force scale rx", 1.0, -10.0, 10.0)
gen.add("absolute_force_scale_ry", double_t, 0, "Absoulte feedback force scale ry", 1.0, -10.0, 10.0)
gen.add("absolute_force_scale_rz", double_t, 0, "Absoulte feedback force scale rz", 1.0, -10.0, 10.0)

exit(gen.generate(PACKAGE, "dynamic_force_scale", "force_scale_param"))
